chapters/revisions.tex

```diff
@@ -1278,7 +1271,7 @@ \subsection{Contributors to the Modelica Language, Version 3.2}\doublelabel{cont
 The global name lookup was proposed by Stefan Vorkoetter.
 
 The support for Unicode was initiated by Rui Gao and Hoyoun Kim.\\
-The \lstinline!homotopy! operator was proposed by Martin Otter, Michael Sielemann
```

Comment:
Starting with non-capital, a simple rewrite is `Martin Otter, ... proposed \lstinline!homotopy!.`

Reply:
Fixed.